ui/app/src/components/FormsEngine/controls/Label.tsx

```
@@ -0,0 +1,30 @@
+
+import type { ControlProps } from '../types';
+import { getPropertyValue } from '../lib/formUtils';
+
+export function Label(props: ControlProps) {
+	const { field } = props;
+
+	// region properties/validations
+	const labelText: string = getPropertyValue(field.properties, 'text') as string;
+	// endregion
+
+	return <span dangerouslySetInnerHTML={{ __html: labelText }} />;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)